### PR TITLE
The Great Deduplication (50%)

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 57 functions (+1.03%) on `us`.

File: src/world/area_isk/isk_05/97DAD0.c [hunk 1]
```diff
@@ -44,3 +46,4 @@ INCLUDE_ASM(s32, "world/area_isk/isk_05/97DAD0", func_80241D44_97F814);
 INCLUDE_ASM(s32, "world/area_isk/isk_05/97DAD0", func_80241DF8_97F8C8);
 
 INCLUDE_ASM(s32, "world/area_isk/isk_05/97DAD0", func_80241E24_97F8F4);
+
```
Comment (nanaian) [inline src/world/area_isk/isk_05/97DAD0.c:49]: Newline at EOF

File: src/world/common/UnkNpcAIFunc39.inc.c [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    Npc* npc = get_npc_unsafe(enemy->npcID);
+    f32 posX, posY, posZ, posW;
+    f32 sp34;
+
+    if (npc->moveSpeed > 0.0) {
+        posX = npc->pos.x;
+        posY = npc->pos.y;
+        posZ = npc->pos.z;
+        if (npc_test_move_simple_with_slipping(npc->unk_80, &posX, &posY, &posZ, npc->moveSpeed, npc->yaw, 
+                                               npc->collisionHeight, npc->collisionRadius) != 0)
```
Comment (nanaian) [inline src/world/common/UnkNpcAIFunc39.inc.c:16]: `!= 0` - this returns a bool, right?

File: src/world/common/UnkNpcAIFunc39.inc.c [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        sp34 = 100.0f;
+        posZ = npc->pos.z;
+        posW = 1000.0f;
+        npc_raycast_down_sides(npc->unk_80, &posX, &sp34, &posZ, &posW);
+
+        posX = npc->pos.x;
+        posY = npc->pos.y + 13.0;
+        posZ = npc->pos.z;
+        posW = fabsf(npc->jumpVelocity) + 16.0;
+        if (npc_raycast_down_sides(npc->unk_80, &posX, &posY, &posZ, &posW) != 0 && posW <= fabsf(npc->jumpVelocity) + 13.0) {
```
Comment (nanaian) [inline src/world/common/UnkNpcAIFunc39.inc.c:35]: " wrt !=0

**Review by nanaian — CHANGES_REQUESTED** [review]

File: src/world/area_isk/isk_05/97DAD0.c (hunk 1 above)
Comment (ethteck) [reply src/world/area_isk/isk_05/97DAD0.c:49]: It has one. it didn't used to

**Pull request merged**